src/cryptopunks.ts

```diff
@@ -415,9 +415,6 @@ export function handlePunkOffered(event: PunkOffered): void {
     );
   }
   ask.from = account.id;
-  if (!ask) {
-    ask.open = true;
-  }

```

Comment:
Can you explain why this change is necessary? (I assume rows 516-518 are deleted for the same reason)

Reply:
Actually we are going to delete the whole cryptopunks soon so I'm resolving this as it is not really relevant.